checkstyle.xml

```diff
@@ -26,25 +26,25 @@
   <property name="localeLanguage" value="en" />
 
   <!-- Checks that a package-info.java file exists for each package. -->
-  <!-- See http://checkstyle.sourceforge.net/config_javadoc.html -->
+  <!-- See https://checkstyle.org/config_javadoc.html -->
```

Comment:
Do you mean https://checkstyle.org/config_javadoc.html#JavadocPackage ?

Reply:
Seems better, changed.